frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -120,39 +120,41 @@ const UPDATE_AOE = gql`
 `;
 
 interface AreYouSureProps {
-  patientName: string;
+  cancelText: string;
+  continueText: string;
   cancelHandler: () => void;
   continueHandler: () => void;
 }
 
 const AreYouSure: React.FC<AreYouSureProps> = ({
-  patientName,
   cancelHandler,
+  cancelText,
   continueHandler,
+  continueText,
+  children,
 }) => (
   <Modal
     isOpen={true}
     style={{
       content: {
         top: "50%",
         left: "50%",
+        bottom: "auto",
```

Comment:
Ah, this is what kept it from auto-sizing. @sharmaneil I think yow were having trouble with your dialogs being too tall as well, this should fix it.

Reply:
After you hopped off that call I was chatting with @sharmaneil about the need for a more unified set of modals. A modal audit and unified style guide would be very helpful

Reply:
Also all the style props can be in classes, I started using them instead for later dialogs. In general I think we can have a common background for all dialogs but it always seems the size and shape of the dialogs themselves are unique. If we do the `OkayCancelDialog` that could possibly run into sizing conflicts but we could add a `className` prop to the dialog so it could be styled.